GliaWidgets/ViewController/Survey/SurveyViewController.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+
+extension Survey {
+    final class ViewController: UIViewController {
+
+        struct Props {
+            let header: String
+            var questionsProps: [QuestionPropsProtocol]
+            var submit: (Props) -> Void
+            var cancell: () -> Void
+
+            init(
+                header: String = "",
+                props: [QuestionPropsProtocol] = [],
+                submit: @escaping (Props) -> Void = { _ in },
+                cancel: @escaping () -> Void = {}
+            ) {
+                self.header = header
+                self.questionsProps = props
+                self.submit = submit
+                self.cancell = cancel
+            }
+        }
+
+        var props: Props {
+            didSet { render() }
+        }
+
+        required init?(coder: NSCoder) { preconditionFailure() }
+
+        init(props: Props = .init()) {
+            self.props = props
+            super.init(nibName: nil, bundle: nil)
+            if #available(iOS 13.0, *) {
```

Comment:
What if the survey only has 1 question, the presentation style will still take a whole screen of space?
I kind of imagined it being more like alert, with custom configurable dimmed background color over the view, and then content being subview on that survey view, only taking up as much space as needed, but not more than screen height * 0.9 or smth. Then disabling the ios 13 modal presentation style alltogether.